alfred-telemetry-solr/alfred-telemetry-solr6/src/main/java/eu/xenit/alfred/telemetry/solr/monitoring/binder/SolrSnapshootMetrics.java

```diff
@@ -4,95 +4,92 @@
 import io.micrometer.core.instrument.MeterRegistry;
 import io.micrometer.core.instrument.Tags;
 import io.micrometer.core.instrument.binder.MeterBinder;
-import java.lang.reflect.Field;
-import java.text.ParseException;
-import java.text.SimpleDateFormat;
-import java.util.Map;
-import java.util.TimeZone;
 import org.alfresco.solr.AlfrescoCoreAdminHandler;
+import org.apache.solr.common.params.MapSolrParams;
+import org.apache.solr.common.params.SolrParams;
 import org.apache.solr.common.util.NamedList;
 import org.apache.solr.core.SolrCore;
 import org.apache.solr.handler.ReplicationHandler;
+import org.apache.solr.handler.RequestHandlerBase;
+import org.apache.solr.request.LocalSolrQueryRequest;
 import org.apache.solr.request.SolrRequestHandler;
+import org.apache.solr.response.SolrQueryResponse;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Map;
+import java.util.TimeZone;
+
 public class SolrSnapshootMetrics implements MeterBinder {
```

Comment:
We need to correct the name to SolrSnapshotMetrics

Reply:
Nope, the terminology in similar classes refers to it as a `snapshoot`, which means the action of taking a snapshot.
https://github.com/apache/solr/blob/26195c82493422cb9d6d4bdf9d4452046e7b3f67/solr/core/src/java/org/apache/solr/handler/ReplicationHandler.java#L231